LSP-vtsls.sublime-commands

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "caption": "Preferences: LSP-vtsls Settings",
+        "command": "edit_settings",
+        "args": {
+            "base_file": "${packages}/LSP-vtsls/LSP-vtsls.sublime-settings",
+            "default": "// Settings in here override those in \"LSP-vtsls/LSP-vtsls.sublime-settings\"\n\n{\n\t$0\n}\n"
+        }
+    },
+    {
+        "caption": "LSP-vtsls: Organize Imports",
+        "command": "lsp_typescript_execute",
```

Comment:
Create a new command `lsp_vtsls_execute`  and use it instead of `lsp_typescript_execute` (in all places you used).`lsp_typescript_execute` comes from LSP-typescript